presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -97,6 +97,50 @@ Property Name                                        Description
 
 ==================================================== ============================================================
 
+To set up a Nessie instance locally using docker image, see `Setting up Nessie <https://projectnessie.org/try/docker/>`_. Once the docker instance is up and running, you should see logs similar to the following example:
+
+.. code-block:: none
+
+    2023-09-05 13:11:37,905 INFO  [io.quarkus] (main) nessie-quarkus 0.69.0 on JVM (powered by Quarkus 3.2.4.Final) started in 1.921s. Listening on: http://0.0.0.0:19120
+    2023-09-05 13:11:37,906 INFO  [io.quarkus] (main) Profile prod activated.
+    2023-09-05 13:11:37,906 INFO  [io.quarkus] (main) Installed features: [agroal, amazon-dynamodb, cassandra-client, cdi, google-cloud-bigtable, hibernate-validator, jdbc-postgresql, logging-sentry, micrometer, mongodb-client, narayana-jta, oidc, opentelemetry, reactive-routes, resteasy, resteasy-jackson, security, security-properties-file, smallrye-context-propagation, smallrye-health, smallrye-openapi, swagger-ui, vertx]
+
+
+If log messages related to Nessie's OpenTelemetry collector appear similar to the following example:
+
+.. code-block:: none
+
+    2023-08-27 11:10:02,492 INFO  [io.qua.htt.access-log] (executor-thread-1) 172.17.0.1 - - [27/Aug/2023:11:10:02 +0000] "GET /api/v1/config HTTP/1.1" 200 62
+    2023-08-27 11:10:05,007 SEVERE [io.ope.exp.int.grp.OkHttpGrpcExporter] (OkHttp http://localhost:4317/...) Failed to export spans. The request could not be executed. Full error message: Failed to connect to localhost/127.0.0.1:4317
+
+you can disable OpenTelemetry using the configuration option ``quarkus.otel.sdk.disabled=true``.
+For example, start the docker image using the following command::
+ ``docker run -p 19120:19120 -e QUARKUS_OTEL_SDK_DISABLED=true ghcr.io/projectnessie/nessie``.
```

Comment:
```suggestion
``docker run -p 19120:19120 -e QUARKUS_OTEL_SDK_DISABLED=true ghcr.io/projectnessie/nessie``.
```

Reply:
These two comments are intended to fix a small formatting problem I might have introduced with my previous comments. My apologies about that. 

First screenshot is how the page looks building locally from the current file
<img width="818" alt="Screenshot 2023-09-06 at 5 40 44 PM" src="https://github.com/prestodb/presto/assets/7013443/b7e02fad-99dd-4794-abf1-0be799e0be5e">
with `` marks before and after, and a period at the end of the command we probably don't want the user to include in the command.  :)

Second screenshot is how the page looks building locally with these two comments' edits: 
<img width="806" alt="Screenshot 2023-09-06 at 5 39 36 PM" src="https://github.com/prestodb/presto/assets/7013443/32c483dd-b168-4b96-966b-beb048d9dbac">

Reply:
Thank you for pointing this out. Have taken care.